.github/workflows/lint/Dockerfile

```diff
@@ -0,0 +1,21 @@
+# Development
+FROM ubuntu:18.04
+LABEL maintainer="Kevin Griffin <griffin.kev@gmail.com>"
```

Comment:
Given these containers are now cached in GitHub packages we should make this maintainer Hyperledger